openstf/pipeline/create_forecast_sklearn.py

```diff
@@ -1,52 +1,55 @@
 # SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
 #
 # SPDX-License-Identifier: MPL-2.0
+from typing import Union
 from datetime import datetime, timedelta, timezone
+from pathlib import Path
 
 import pandas as pd
 import structlog
-
-# from ktpbase.config.config import ConfigManager
+from sklearn.base import RegressorMixin
 
 from openstf.validation import validation
 from openstf.feature_engineering.feature_applicator import (
     OperationalPredictFeatureApplicator,
 )
 from openstf.model.confidence_interval_applicator import ConfidenceIntervalApplicator
-from openstf.preprocessing import preprocessing
+
 from openstf.model.serializer import PersistentStorageSerializer
 from openstf.postprocessing.postprocessing import (
     add_prediction_job_properties_to_forecast,
 )
 from openstf.model.fallback import generate_fallback
 
 
-# TODO add loading of model to task
-# # Load most recent model for the given pid
-# model = PersistentStorageSerializer(
-#     trained_models_folder=MODEL_LOCATION
-# ).load_model(pid=pj["id"])
+def create_forecast_pipeline(
+    pj: dict, input_data: pd.DataFrame, trained_models_folder: Union[str, Path]
+) -> pd.DataFrame:
+    """Midle level pipeline that takes care of all persistent storage dependencies
```

Comment:
The title of the docstring should be a bit shorter I think. More detailed explanation goes in the body I guess. It seems also a bit inconsistent with the other pipeline docstrings. Why is this a middle level pipeline and some others not, what does middle  mean in this context? Perhaps something like this makes more sense:?

```python
    """Create forecast pipeline.
        
        This is the top-level pipeline which included loading the most recent model 
        for the given prediction job.
    """
```